worker/test/src/RTC/RTCP/TestXr.cpp

```diff
@@ -57,7 +57,7 @@ SCENARIO("RTCP XrDelaySinceLastRt parsing", "[parser][rtcp][xr-dlrr]")
 		packet1->Serialize(bufferPacket1);
 
 		// Create a new packet out of the external buffer.
-		auto packet2 = ExtendedReportPacket::Parse(bufferPacket1, packet1->GetSize());
+		auto packet2 = std::unique_ptr<ExtendedReportPacket>(ExtendedReportPacket::Parse(bufferPacket1, packet1->GetSize()));
```

Comment:
So `Parse` returns heap-allocated data structure. I think we should change `auto` to `auto*` to make that clearer and just call `delete` at the end for consistency.

We can also use `std::unique_ptr`, which I guess will be beneficial due to not leaking memory even when test fails, but then we might want to use it in all other tests too consistently.